Vikta-API-testing-sample/src/test/java/com/test/api/category/CategoryTest.java

```diff
@@ -28,17 +31,29 @@ public void verifyGetCategory() {
                 equalTo(expectedTitle));
     }
 
+    /**
+     * Test with wrong id given
+     */
+    @Test
+    public void verifyGetCategoryWithUnexpectedId() {
```

Comment:
create a parameterized tests for negative cases. for each available action create separate test method that would accept several negative cases to assert. Use documentation to check boundary values and prepare cases based on that